client/agent-opentracing/src/main/resources/apmrules/org/apache/apm-apache-http-client.btm

```diff
@@ -35,3 +35,14 @@ IF hasSpan()
 DO
   finishSpan();
 ENDRULE
+
+RULE org.apache.http.client(4) Apache HttpClient Producer Sync Exception
+CLASS ^org.apache.http.impl.client.CloseableHttpClient
+METHOD doExecute
+HELPER org.hawkular.apm.agent.opentracing.OpenTracingManager
+AT EXCEPTION EXIT
+IF hasSpan()
+DO
+  getSpan().setTag("error","true").setTag("http.status_code",String.valueOf(503));
```

Comment:
I would take it out, for the reason what you sad that is no status code it's on different network layer no?  

It would be good to create a jira to add log with exceptions after it is in model :+1: . 

Tho things:
* is it possible to set a boolean for `error` tag?
* a test would be good for it